core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/plans.scala

```diff
@@ -133,6 +133,24 @@ abstract class Plan[PlanType <: Plan[PlanType]] extends TreeNode[PlanType] {
       case other => Nil
     }.toSeq
   }
+
+  /** Returns all expressions present in this query plan and its descendants */
+  def nestedExpressions: Seq[Expression] = {
```

Comment:
why do we need to modify a widely-used class with the method that's used only once?